sim/druid/enrage.go

```diff
@@ -11,30 +11,22 @@ func (druid *Druid) registerEnrageSpell() {
 	actionID := core.ActionID{SpellID: 5229}
 	rageMetrics := druid.NewRageMetrics(actionID)
 
-	instantRage := []float64{20, 24, 27, 30}[druid.Talents.Intensity]
-
-	dmgBonus := 0.05 * float64(druid.Talents.KingOfTheJungle)
-
-	t10_4p := druid.HasSetBonus(ItemSetLasherweaveBattlegear, 4)
+	instantRage := []float64{0, 5, 10}[druid.Talents.ImprovedEnrage]
```

Comment:
The [SoD tuning aura](https://www.wowhead.com/classic/spell=436895/s03-tuning-and-overrides-passive-druid) for druid makes a few changes to Enrage which essentially moves half of the rage to being instant. It gives 20 instantly and makes it give only 1 per tick, if I'm reading correctly.

Improved Enrage talent still adds additional up front rage.